libs/component-library/src/lib/progress-bar/progressBar.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { FxBox, FxBoxProps } from '../box/box';
+import { configureEaseInOutLayoutAnimation } from '../utils/animations';
+import { WINDOW_WIDTH } from '../utils/constants';
+
+export type FxProgressBarProps = {
+  progress: number;
+  width?: number;
+  total?: number;
+  disabled?: boolean;
+} & FxBoxProps;
+
+const FxProgressBar = ({
+  width,
+  progress,
+  height = 4,
+  total = 100,
+  ...rest
+}: FxProgressBarProps) => {
+  const _width = width || WINDOW_WIDTH;
+  const _progress = Math.min(Math.max(0, progress), total);
+  const progressWidth = Math.round(_width * (_progress / total));
+
+  React.useEffect(() => {
+    configureEaseInOutLayoutAnimation();
```

Comment:
I've noticed when progress is `0` and then we press `Increase Progress`, the animation isn't running.

Reply:
Seems to be working for me with layout now: 
![Aug-23-2022 16-38-25](https://user-images.githubusercontent.com/17250443/186201363-f09c1355-2ba2-4836-80f4-df3c009b297b.gif)